app/javascript/article-form/articleForm.jsx

```diff
@@ -343,6 +355,28 @@ export default class ArticleForm extends Component {
           helpPosition={helpPosition}
           version={version}
         />
+        {this.state.showModal && (
+          <Modal title="You have unsaved changes" onClose={this.toggleModal}>
+            <div>
+              <p>
+                You've made changes to your post. Do you want to navigate to
+                leave this page?
+              </p>
+              <ButtonGroup className="crayons-article-form__modal__button-group">
```

Comment:
We usually use `<ButtonGroup />` component for scenarios where two buttons are meant to look like a segmented control (so kinda "touching" themselves with one edge... im terrible at describing it :D), here's an example: 

![image](https://user-images.githubusercontent.com/108287/105830412-c304cd80-5fc5-11eb-9720-dd3d5603791d.png)

For scenarios like we have in this modal, we don't use this component, we just place two buttons next to each other.

---

Regarding my suggestion in previous comment about utility-first classes... If we replace that component with classic `div`, we can simply apply `pt-4` class for `padding-top` and it should do the job. For spacing between button we can either do one of the following things:

```
<div className="pt-4 flex gap-2">
    <Button>...
    <Button>...
</div>
```

or

```
<div className="pt-4">
    <Button className="mr-2">...
    <Button>...
</div>
```

Reply:
Ah! Great! I was wondering why the 'ButtonGroup' needed to be modified 😅